docker/tpu/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM google/cloud-sdk:slim
+
+# Build args.
+ARG GITHUB_REF=refs/heads/master
+ARG TEST_IMAGE=0
+
+# TODO: This Dockerfile installs pytorch/xla 3.6 wheels. There are also 3.7
+# wheels available; see below.
+ENV PYTHON_VERSION=3.6
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+         build-essential \
+         cmake \
+         git \
+         curl \
+         ca-certificates
+
+# Install conda and python.
+RUN curl -o ~/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh  && \
+    chmod +x ~/miniconda.sh && \
+    #~/miniconda.sh -b -p /home/containeruser/conda && \
+    ~/miniconda.sh -b && \
+    rm ~/miniconda.sh
+
+ENV PATH=/root/miniconda3/bin:$PATH
+
+RUN conda create -y --name container python=$PYTHON_VERSION
+
+# Run the rest of commands within the new conda env.
+SHELL ["conda", "run", "-n", "container", "/bin/bash", "-c"]
+RUN conda install -y python=$PYTHON_VERSION mkl
+
+RUN pip uninstall -y torch
+# TODO: Python 3.7 wheels are available. Replace cp36-cp36m with cp37-cp37m.
+RUN gsutil cp 'gs://tpu-pytorch/wheels/torch-nightly-cp36-cp36m-linux_x86_64.whl' .
+RUN gsutil cp 'gs://tpu-pytorch/wheels/torch_xla-nightly-cp36-cp36m-linux_x86_64.whl' .
+RUN gsutil cp 'gs://tpu-pytorch/wheels/torchvision-nightly-cp36-cp36m-linux_x86_64.whl' .
+RUN pip install 'torch-nightly-cp36-cp36m-linux_x86_64.whl'
```

Comment:
shall we rather use some stable versions then nightly? will users use stable or nightly?

Reply:
Unfortunately right now we recommend `nightly` to users. We have plans in the near future to have a recommended stable version and once that happens we would want to switch this part to point to that version